dashboard/config/linux/upstream-apparmor-kasan.config

```diff
@@ -203,7 +203,7 @@ CONFIG_CGROUPS=y
 CONFIG_PAGE_COUNTER=y
 # CONFIG_CGROUP_FAVOR_DYNMODS is not set
 CONFIG_MEMCG=y
-CONFIG_MEMCG_KMEM=y
+# CONFIG_MEMCG_V1 is not set
```

Comment:
I think we want this. Some of our kernels enable cgroups v1, and distros will likely to keep them enabled.